thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/datasource/sql/SqlResponseCacheLoader.java

```diff
@@ -209,22 +213,32 @@ public SqlResponseCacheLoader(Map<String, Object> properties) throws Exception {
               File file = new File(fileURI);
               try (Scanner scanner = new Scanner(file)) {
                 String columnNames = scanner.nextLine();
-                while (scanner.hasNextLine()) {
-                  String line = scanner.nextLine();
-                  String[] items = line.split(",");
-                  DateTime dateTime = DateTime.parse(items[0], fmt);
-                  if (dateTime.isAfter(maxDateTime)) {
-                    maxDateTime = dateTime;
+
+                // For demo datasets, we want to show some sample plots for recent time period.
+                // We do this by sliding the historical static data stored in [daily|hourly].csv
+                // to the current timestamp.
+                if (isDemoDataset(tableName)) {
```

Comment:
Shall we put this logic into `MockThirdEyeDataSource` and use that instead? That one is built for demo purposes. Feels like a hack to put it in the `SqlResponseCacheLoader`

Reply:
That's a good idea. I will see how I can move it under the MockDataSource.